packages/lab/src/slider/Slider.tsx

```diff
@@ -1,70 +1,42 @@
-import { makePrefixer, useControlled } from "@salt-ds/core";
+import { Label, makePrefixer, useControlled } from "@salt-ds/core";
 import { clsx } from "clsx";
-import {
-  CSSProperties,
-  forwardRef,
-  HTMLAttributes,
-  useMemo,
-  useRef,
-} from "react";
-import { SliderHandle } from "./internal/SliderHandle";
-import { SliderMarkLabels } from "./internal/SliderMarkLabels";
-import { SliderRail } from "./internal/SliderRail";
-import { SliderMark, SliderRailMarks } from "./internal/SliderRailMarks";
-import { SliderSelection } from "./internal/SliderSelection";
-import { createHandleStyles, createTrackStyle } from "./internal/styles";
-import { useSliderKeyDown } from "./internal/useSliderKeyDown";
-import { useSliderMouseDown } from "./internal/useSliderMouseDown";
-import { useValueUpdater } from "./internal/utils";
-import { SliderChangeHandler, SliderValue } from "./types";
-
+import { forwardRef, HTMLAttributes } from "react";
 import { useWindow } from "@salt-ds/window";
 import { useComponentCssInjection } from "@salt-ds/styles";
+import { SliderTrack, SliderMarks, SliderContext } from "./internal";
 
 import sliderCss from "./Slider.css";
+import { SliderChangeHandler } from "./types";
 
 const withBaseName = makePrefixer("saltSlider");
 
 const defaultMin = 0;
-const defaultMax = 100;
+const defaultMax = 10;
 const defaultStep = 1;
 
 export interface SliderProps
   extends Omit<HTMLAttributes<HTMLDivElement>, "onChange" | "defaultValue"> {
-  label?: string;
   min?: number;
   max?: number;
   step?: number;
-  pageStep?: number;
-  value?: SliderValue;
-  defaultValue?: SliderValue;
-  pushable?: boolean;
-  pushDistance?: number;
-  disabled?: boolean;
+  value?: number;
+  defaultValue?: number;
+  labels?: "inline" | "bottom" | "marks";
   onChange?: SliderChangeHandler;
-  marks?: SliderMark[];
-  hideMarks?: boolean;
-  hideMarkLabels?: boolean;
 }
 
 export const Slider = forwardRef<HTMLDivElement, SliderProps>(function Slider(
   {
     min = defaultMin,
     max = defaultMax,
     step = defaultStep,
-    pageStep = step,
     value: valueProp,
     defaultValue = defaultMin,
     onChange,
-    label,
     className,
-    pushable,
-    pushDistance = 0,
-    disabled,
-    marks,
-    hideMarks,
-    hideMarkLabels,
-    ...restProps
+    ["aria-label"]: ariaLabel,
```

Comment:
de-structure the aria label so this can be attached to the slide thumb - still looking into accessibility with form fields - pr to follow with more accessibility